packages/backend/src/peripherals/database/migrations/020_events.ts

```diff
@@ -0,0 +1,32 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server. 
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
+  await knex.schema.createTable('events', function (table) {
```

Comment:
Add indexes for `unix_timestamp`, `project_id` and `time_span`

Reply:
The index created in private key works for single columns, I tested it by sql analyze